backend/server/serverdb.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+require("dotenv").config({ path: "./config.env" });
+const port = process.env.PORT || 5000;
+app.use(cors());
+app.use(express.json());
+app.use(require("./routes/record"));
+// get driver connection
+const dbo = require("./db/conn");
+ 
+app.listen(port, () => {

```

Comment:
Should this be in server.js

Reply:
sure, should we merge the files and put it into the server folder?